static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,144 @@
+@import 'svg';
+$news-main-1-dark: #003d70;
+
+
+
+.interactive-embed  {
```

Comment:
Is `interactive-embed` the right class here now? Don't think it's an interactive anymore?